routes/index.js

```diff
@@ -4,11 +4,12 @@ var Product = require('../models/product');
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
-  res.render('index', { products: /* ??? */ });
+  res.render('index', { products: Product.all });
 });
 
 router.post('/cart', function(req, res, next) {
-  // How do we add an item to the cart?
+  new Product(req.body.id) - 1;
```

Comment:
You're on the right track here with wanting the id from the body, but you have to use parseInt to make sure it returns as a number and not a string. You're also not creating a new Product in this route, but rather using the id from the body to then push it into the cart.